File: problemas/ad-hoc/2388/2388.r
input <- file('stdin', 'r')

n <- as.integer(readLines(input, n=1))

resposta <- 0
for(i in 1:n) {
    entrada <- as.integer(strsplit(readLines(input, n = 1), ' ')[[1]])
    resposta <- resposta + entrada[1] * entrada[2]
}

write(paste(resposta), '')

File: problemas/ad-hoc/2388/2388.c
#include <stdio.h>

int main()
{
    int n, t, v;

    scanf("%d\n", &n);

    int resposta = 0;
    for (int i = 0; i < n; ++i)
    {
        scanf("%d %d\n", &t, &v);
        resposta += t * v;
    }
    printf("%d\n", resposta);

    return 0;
}